docs/how-to-guides/develop-a-tool/create-your-own-custom-strong-type-connection.md

```diff
@@ -0,0 +1,100 @@
+# Create and Use Your Own Custom Strong Type Connection
+Connections provide a secure method for managing credentials for external APIs and data sources in Prompt flow. This guide explains how to create and use a custom strong type connection.
+
+## What is a Custom Strong Type Connection?
```

Comment:
use `###` here

Reply:
Build Doc CI does not allow skipping using ###, so we can only use ## here